docs/references/vod/patch.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: PATCH updating storage
+---
+
+
+`PATCH /asset/:id -d {"storage": {"ipfs": {}}` can be used to modify any `asset` that was stored with On Demand. It is a single endpoint that can update only the following mutable properties of an asset:
+
+- `name`
+- `storage`
+- `meta`
+
+
+
+## Update Stored Asset
+
+> **NOTE:** When updating the asset information, the `storage` object must be included in the body of the request
+
+## Updating Asset Name
+
+This will allow the name of the stored asset in Livepeer Studio to be changed.
+
+### Request
+
+```bash
+curl -X PATCH https://livepeer.studio/api/asset/{id} \
+-H 'content-type: application/json' \
+-H 'authorization: Bearer {api-key}' \
+-d {
+  "name": "New name"
+  "storage": {"ipfs": {}}
+  }
+```
+
+## Updating Storage
+
+This allows to add additional storage location of where the assets are to be stored.
+
+>**NOTE:** 
+- Currently assets cannot be deleted from storage if stored on Livepeer Studio. 
+- Only IPFS is support as an additional storge option at the moment
+
+
+### Request
+
+```bash
+curl -X PATCH https://livepeer.studio/api/asset/{id} \
+-H 'content-type: application/json' \
+-H 'authorization: Bearer {api-key}' \
+-d {"storage": {"ipfs": {}}}
```

Comment:
We should add some instructions about the `nftMetadata` field, which can be used to customize the asset NFT metadata.

Reply:
Need to know what are the metadata fields and how to customize it.

Reply:
These are the default metadata templates: https://github.com/livepeer/task-runner/blob/4d1ae1768c01e979969bf7f00b81267db05e299c/task/export.go#L154
(default is the first one)

But there is no enforced schema here. Users can set anything they want. Each marketplace has their own standard, but they generally follow OpenSea's: https://docs.opensea.io/docs/metadata-standards
These are Lens, compatible and strongly based on OpenSea: https://docs.lens.xyz/docs/metadata-standards

All those marketplaces also make sure to be compatible with one of the real "NFT standards":
 - [ERC-721](https://eips.ethereum.org/EIPS/eip-721#specification) (basically `name`, `description` and `image`)
 - [ERC-1155](https://eips.ethereum.org/EIPS/eip-1155#metadata) (`name`, `description`, `decimals`, `image` and `properties`)

Also notice that the `attributes` field, used by OpenSea, Lens, etc, is not present in any stoken standard. It was just made up by OpenSea because ERC-721 didn't have it and kinda became the standard nowadays... We use the `properties` field by default, but most apps actually use `attributes` (including Lens).

We do not have any NFT metadata standards of our own. It is acceptable to say that users should just customize their NFT metadata following any standard that they want.